.github/CONTRIBUTING.md

```diff
@@ -47,35 +82,6 @@ To run the example app on iOS:
 yarn example ios
 ```
 
-By default, the example is configured to build with the old architecture. To run the example with the new architecture, you can do the following:
```

Comment:
Wasn't sure if anyone ever tested this, and didn't want to have erroneous/untested directions in here

Reply:
I have not.

Reply:
Figured we can add it back whenever we do turbo modules, same vein.